README.md

```diff
@@ -41,13 +41,19 @@ If you would like to contribute multi-intent utterences, please add a `+` to ind
 
 ### Where do I put intents that fall into multiple domains?
 
-Please create a new folder for mixed domains, for example: “mixed-smalltalk+weather”
+Please create a new folder for mixed domains, for example: “mixed-smalltalk+transport”
 
 ### What about training data that’s not in English? 
 
 Currently, we are unable to evaluate the quality of all language contributions, and therefore, during the initial phase we can only accept English training data to the repository.
 However, we understand that the Rasa community is a global one, and in the long-term we would like to find a solution for this in collaboration with the community. 
 
+### Why do I need to remove entities from my training data? 
+
+We would like to make the training data as easy as possible to adopt to new training models and annotating entities is personalized and highly dependant on your bots purpose. Therefore, we will first focus on collecting training data that only includes intents.
```

Comment:
I would skip the "personalized". Dependent on the bot's purpose is fine.
Typo: "dependant on your _bot's_..."